cli/lightning-cli.c

```diff
@@ -197,8 +201,8 @@ static void human_help(char *buffer, const jsmntok_t *result)
 	for (i = 0; i < tal_count(help); i++) {
 		const jsmntok_t *command;
 		command = json_get_member(buffer, help[i], "command");
-		printf("%.*s\n\n",
-		       command->end - command->start, buffer + command->start);
+		human_readable(buffer, command, '\n');
+		printf("\n");
 	}
```

Comment:
human_readable() is used in other contexts, and this will break it.  In particular, the summary plugin.

I think this replacement needs to occur inline:

```
static void replace_char(char *buf, size_t len, char old, char new)
{
     char *p = buf, *end = buf + len;
     while ((p = memchr(p, old, end - p)) != NULL)
          *p = new;
}
```

Then simply call `replace_char(buffer + command->start, command->end - command_start, '|', '\n')` here.